### PR TITLE
Feature: Jetpack AI Logo generator

### PR DESCRIPTION
**Pull request by CGastrell**
This PR stands as a feature branch for the Jetpack AI Logo generator

Related to p1HpG7-qjH-p2

## Proposed Changes

* Adds the Jetpack AI Logo Generator feature to the `jetpack-ai-calypso` package.
* Adds the feature to the quick links section of My Home.

## Testing Instructions

For feature testing, see pedxs5-n0-p2.
The feature is contained in a new package and only imported in the quick links section of My Home, so check if there is no build errors or performance issues and nothing else obviously breaks.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/jetpack-ai-calypso/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/jetpack-ai-calypso",
-	"version": "1.0.0",
+	"version": "2.0.0",
 	"description": "Jetpack AI utilities available to Calypso.",
 	"calypso:src": "src/index.js",
```
Comment (jsnajdr) [inline packages/jetpack-ai-calypso/package.json]: This points to `src/index.js`, but the real file name is `src/index.ts`. I suspect that this is causing the issues that forced you to add `SKIP_TSC=true` and remove the `yarn run clean` calls.

**Review by jsnajdr — APPROVED** [review]
Looks good from a frameworks point of view, but please fix the `calypso:src` reference.

File: packages/jetpack-ai-calypso/package.json (hunk 1 above)
Comment (dhasilva) [reply packages/jetpack-ai-calypso/package.json]: Good catch! Thank you.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11138636

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @CGastrell for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11138636).

**Pull request merged**